--- a/smartd.conf
+++ b/smartd.conf
@@ -1,9 +1,8 @@
-# Sample configuration file for smartd.  See man smartd.conf.
+# /etc/smartd.conf
+# Configuration file for smartd. Use "man smartd.conf" for more information.
 
 # Home page is: http://smartmontools.sourceforge.net
 
-# $Id: smartd.conf,v 1.45 2006/11/12 23:39:04 dpgilbert Exp $
-
 # smartd will re-read the configuration file if it receives a HUP
 # signal
 
