--- a/smartd.cpp
+++ b/smartd.cpp
@@ -1038,7 +1038,7 @@ static void MailWarning(const dev_config
              "  NIS domain: %s\n\n"
              "The following warning/error was logged by the smartd daemon:\n\n"
              "%s\n\n"
-             "For details see host's SYSLOG (default: /var/log/messages).\n\n"
+             "For details see host's SYSLOG (default: /var/log/syslog).\n\n"
              "%s%s%s",
 	     hostname, domainname, nisdomain, message, further, original, additional);
   exportenv(environ_strings[10], "SMARTD_FULLMESSAGE", fullmessage);
